### DISCUSSION TITLE
First execution hellorails-web-1 hellorails-css-1 hellorails-js-1

### DISCUSSION THREAD

**Discussion by loweffort-alt** [nickjj/docker-rails-example#78] [Q&A]
Hey @nickjj, just wanted to drop by from your YouTube video - awesome stuff!

So, when I'm running `docker compose up` in Git Bash, these containers hellorails-web-1, hellorails-js-1, and hellorails-css-1 don't seem to kick off. I followed all the steps in the Readme file, but when I run `./run rails db:setup`, it produces this error **'service "web" is not running'** and I'm seeing the same logs for those containers:
```
yarn run v1.22.22
$ ./run yarn:build
2024-04-05 21:40:10 /usr/bin/env: ‘bash\r’: No such file or directory
2024-04-05 21:40:10 /usr/bin/env: use -[v]S to pass options in shebang lines
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

2024-04-05 21:40:10 hellorails-web-1       | /usr/bin/env: ‘bash\r’: No such file or directory
2024-04-05 21:40:10 hellorails-web-1       | /usr/bin/env: use -[v]S to pass options in shebang lines

yarn run v1.22.22
$ ./run yarn:build:css
2024-04-05 21:40:09 /usr/bin/env: ‘bash\r’: No such file or directory
2024-04-05 21:40:09 /usr/bin/env: use -[v]S to pass options in shebang lines
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Thanks in advance for any advice you can offer! If you need more details or there's anything else I can provide to help troubleshoot this issue, please let me know. Cheers!

**Comment by nickjj** [comment]
Hi,

It looks like `bash` isn't available in the environment where you're running these commands, or another possibility is the files have been saved with Windows line endings instead of Unix which I've seen cause similar types of errors.

I never used git bash so I'm not sure what type of Linux environment it sets up. Is using WSL available?
